hack/scripts/payload/DeleteMicroVM.json

```diff
@@ -1,4 +1,5 @@
 {
   "id": "mvm1",
-  "namespace": "ns1"
+  "namespace": "ns1",
+  "uid": "b1c0aa3c-ebba-498a-a7c7-903d91758d83"
```

Comment:
should the other 2 be removed?

Reply:
yes

Reply:
But that "tooling" has to be refined after Find.